FSharp-4.5/FS-1053-span.md

```diff
@@ -336,6 +336,8 @@ in some situations where capture must be avoided.
 * Cannot be static or instance members of a class or normal struct
 * Cannot be captured by any closure construct (async methods or lambda expressions)
 * Cannot be used as a generic parameter
+    - In F# 9 and higher, this restriction is relaxed if the generic parameter is defined in C# using the `allows ref struct` anti-constraint. F# can instantiate such generics in types and methods with byref-like types. As a few examples, this affects BCL delegate types (`Action<>`,`Func<>`), interfaces (`IEnumerable<>`,`IComparable<>`) and generic arguments with a user-provided accumulator function (`String.string Create<TState>(int length, TState state, SpanAction<char, TState> action)`)
+    - It is impossible to author generic code supporting byref-like types in F#.
```

Comment:
Would it be worth expanding on this here? E.g., to mention that the `allows ref struct` anti-constraint is not automatically propagated in the same way as normal generic constraints, with some examples.

```fsharp
let a : Action<'T> = …
let f (x : 'T) = a.Invoke x // Can 'T be ReadOnlySpan<int> here? Presumably not?
```

```fsharp
let f (g : Action<'T>) (x : 'T) = g.Invoke x // Can 'T be ReadOnlySpan<int> here?
```

Reply:
It is not automatically propagated, this makes sense to expand on - thanks for checking this.